### PR TITLE
feat: add X libraries

### PR DESCRIPTION
**Pull request by lpcalisi**
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: libxcvt.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: libxcvt
+  version: 0.1.2
+  epoch: 0
+  description: "VESA CVT standard timing modeline generation library & utility"
+  copyright:
+    - license: custom
```
Comment (imjasonh) [inline libxcvt.yaml:7]: We can't accept packages that aren't licensed under accepted licenses. Can you give more details about the license for this package?

**Review by imjasonh — CHANGES_REQUESTED** [review]

File: libxcvt.yaml (hunk 1 above)
Comment (lpcalisi) [reply libxcvt.yaml]: sorry, i forgot include it. 

i've added [licence](https://gitlab.freedesktop.org/xorg/lib/libxcvt/-/blob/master/COPYING)

File: libxcvt.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: libxcvt
+  version: 0.1.2
+  epoch: 0
+  description: "VESA CVT standard timing modeline generation library & utility"
+  copyright:
+    - license: MIT
```
Comment (lpcalisi) [inline libxcvt.yaml:7]: [licence](https://gitlab.freedesktop.org/xorg/lib/libxcvt/-/blob/master/COPYING)

File: libxss.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package:
+  name: libxss
+  version: 1.2.4
+  epoch: 0
+  description: "Xlib-based X11 Screen Saver extension client library"
+  copyright:
+    - license: custom:XFREE86
```
Comment (lpcalisi) [inline libxss.yaml:7]: [licence](https://gitlab.freedesktop.org/xorg/lib/libxfont/-/blob/master/COPYING).

The same as [libx11](https://github.com/wolfi-dev/os/blob/main/libx11.yaml#L7)

**Comment by lpcalisi** [comment]
@imjasonh fixed

**Comment by lpcalisi** [comment]
@ajayk

**Pull request closed without merging**